ci/uninstallDevDeps.sh

```
@@ -1,22 +1,22 @@
-if [ -n "$depsToUninstall" ]; then
-  echo "Uninstalling dev dependencies: $depsToUninstall"
-  npm uninstall $depsToUninstall
-else
-  echo "No dev dependencies to uninstall"
-fi
+#!/bin/bash
+
+# Read package exclusion list from arguments
+exclusionList=("$@")
+
+# Convert exclusion list to grep pattern
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)